postgres/datadog_checks/postgres/metrics_cache.py

```diff
@@ -187,7 +187,25 @@ def get_activity_metrics(self, version):
         metrics_data = self.activity_metrics
 
         if metrics_data is None:
-            query = ACTIVITY_QUERY_10 if version >= V10 else ACTIVITY_QUERY_LT_10
+            excluded_aggregations = self.config.activity_metrics_excluded_aggregations
+            if version < V9:
+                excluded_aggregations.append('application_name')
+
+            default_aggregations = ['datname', 'application_name']
```

Comment:
does it make sense to add a user aggregation here as well? we grab that for DBM activity [data](https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/statement_samples.py#L61)

Reply:
Good idea. The risk of churning and cardinality of user should be rather low so it's probably safe to add.